### DISCUSSION TITLE
yank inside indentation

### DISCUSSION THREAD

**Discussion by avidseeker** [vim/vim#16352] [Q&A] [answered]
Python syntax is clean and simple. But unfortunately too clean for Vim tricks like `yi{` which can be done in languages like C.

Is there a way to have yank work in different indentation levels?

**Comment by benknoble** [comment, marked as answer]
Looks like my mailing list comment didn’t copy: I use a plugin https://github.com/michaeljsmith/vim-indent-object/ to provide indent-based text objects. It seems to work well.
